app/controllers/admin/claims_controller.rb

```diff
@@ -4,10 +4,16 @@ class Admin::ClaimsController < Admin::BaseAdminController
   before_action :ensure_service_operator
 
   def index
-    @claims = Claim.includes(:decisions, eligibility: [:claim_school, :current_school]).awaiting_decision.order(:submitted_at)
+    @claims = Claim.approved if params[:status] == 'approved'
+    @claims = Claim.rejected if params[:status] == 'rejected'
```

Comment:
Feels like this should be an `if/else` or `||=` like below.

Reply:
It feels like it but no, the if/else looks really hard to ready

Reply:
It's just that `params[:status]` can't be both `approved` and `rejected`. If `@claims = Claim.approved` it will never become `Claim.rejected`.